test/AWS.Logger.AspNetCore.Tests/TestFilter.cs

```diff
@@ -183,9 +183,9 @@ public void DefaultFilterCheck()
             logger.LogCritical("critical");
 
             Assert.Equal(5, coreLogger.ReceivedMessages.Count);
-            Assert.Contains("warning\r\n", coreLogger.ReceivedMessages);
```

Comment:
If these tests are the same, just with different inputs/outputs, you could use "Theory" to greatly reduce boilerplate.

Reply:
The test have more unique logic in each test case before these Asserts.